### ISSUE TITLE
paranoid don't open and get this error message:

### ISSUE THREAD

**Issue by robsonsilv4** [Antergos/antergos-openbox-setup#13]
** (paranoid.py:8041): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-VKSgiJ6fCO: Conexão recusada
/usr/bin/paranoid.py:20: PyGIWarning: Gtk was imported without specifying a version first. Use gi.require_version('Gtk', '3.0') before import to ensure that the right version gets loaded.
  from gi.repository import Gtk, GObject
Traceback (most recent call last):
  File "/usr/bin/paranoid.py", line 455, in <module>
    g = GUI()
  File "/usr/bin/paranoid.py", line 187, in __init__
    self.radius.set_value(getInt("shadow-radius"))
TypeError: Must be number, not str
